src/test/util_tests.cpp

```diff
@@ -1217,6 +1217,25 @@ BOOST_AUTO_TEST_CASE(test_DirIsWritable)
     fs::remove(tmpdirname);
 }
 
+BOOST_AUTO_TEST_CASE(test_GetRelativePath)
+{
+    fs::path base = fs::temp_directory_path();
```

Comment:
Use `SetDataDir` instead

Reply:
I had that but `datadir` seemed unrelated to this function. I can revert if there is a reason to do that.